collections/attribute/Create certified attribute (internal).bru

```diff
@@ -0,0 +1,30 @@
+meta {
+  name: Create certified attribute (internal)
+  type: http
+  seq: 9
+}
+
+post {
+  url: {{host-attribute}}/internal/certifiedAttributes
+  body: json
+  auth: none
+}
+
+headers {
+  Content-Type: application/json
+  Accept: */*
+  Accept-Encoding: gzip, deflate, br
+  Authorization: {{JWT}}
+  X-Correlation-Id: {{correlation-id}}
+  Content-Type: application/json
+  ~Connection: keep-alive
+}
```

Comment:
Like we did in other PRs, remove all headers except Authorization and X-Correlation-Id